PACKAGING

```diff
@@ -1,37 +1,37 @@
 # vim: syntax=pod
 
 If you read this file _as_is_, just ignore the funny characters you
-see.  It is written in the POD format (see pod/perlpod.pod) which is
+see.  It is written in the POD format (see F<pod/perlpod.pod>) which is
 specifically designed to be readable as is.
 
 =head1 NAME
 
-PACKAGING - notes and best practice for packaging perl 5
+PACKAGING - Notes and best practice for packaging Perl
 
 =head1 SYNOPSIS
 
 This document is aimed at anyone who is producing their own version of
-perl for distribution to other users. It is intended as a collection
+C<perl> for distribution to other users. It is intended as a collection
 of useful tips, advice and best practice, rather than being a complete
-packaging manual. The starting point for installing perl remains
+packaging manual. The starting point for installing C<perl> remains
 F<INSTALL>.
 
 =head1 Customizing test running
 
-A small number of porting tests (those in t/porting) are not well suited
+A small number of porting tests (those in F<t/porting)> are not well suited
```

Comment:
The parenthesis should be outside the `F<>`.

Reply:
Correct, will fix that